static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -150,12 +161,26 @@ const requestBids = (
             () =>
                 new Promise(resolve => {
                     window.pbjs.que.push(() => {
+                        // TODO: Replace with commercial core's API
+                        // recordFirstPrebidStarted();
+                        EventTimer.trigger('prebidStart');
```

Comment:
@ashishpuliyel can you review this I get an error that trigger is not a function, is that the correct way of using the API?
<img width="1009" alt="Screenshot 2021-02-02 at 17 45 41" src="https://user-images.githubusercontent.com/51630004/106624595-8107f880-657e-11eb-97a7-06ba9dcab71b.png">